buildtools/bin2res.cpp

```diff
@@ -39,124 +39,131 @@ struct embed
 
 typedef std::vector<std::string> stringlist;
 
-const int MAX_FILE_SIZE = 256*256*256;		// = 16 MB
+const int MAX_FILE_SIZE = 256 * 256 * 256;		// = 16 MB
 
 
-int main( int argc, char * * argv )
+int main( int argc, char ** argv )
 {
-	if( argc < 2 )
+	if ( argc < 2 )
 	{
 		std::cerr << "Usage:"  << std::endl << "\t" << argv[0] <<
-							" files" << std::endl;
-		return( 1 );
+			  " files" << std::endl;
+		return ( 1 );
 	}
 
 	std::cout << "// Generated by bin2res" << std::endl;
-
 	std::vector<embed *> embedded_data;
-
 	std::cout << "#ifndef _EMBEDDED_RESOURCES_H" << std::endl;
 	std::cout << "#define _EMBEDDED_RESOURCES_H" << std::endl;
-
 	stringlist files;
-	for( int i = 1; i < argc; ++i )
+
+	for ( int i = 1; i < argc; ++i )
 	{
 		files.push_back( std::string( argv[i] ) );
 	}
-	for( stringlist::iterator it = files.begin(); it != files.end(); ++it )
+
+	for ( stringlist::iterator it = files.begin(); it != files.end(); ++it )
 	{
 		std::ifstream f( it->c_str(), std::ios::binary );
-		if( f.fail() )
+
+		if ( f.fail() )
 		{
 			std::cerr << "Cannot open file " << *it <<
-						", ignoring it" << std::endl;
+				  ", ignoring it" << std::endl;
 			continue;
 		}
+
 		f.seekg( 0, std::ios::end );
 		int fsize = f.tellg();
 		f.seekg( 0 );
-		if( fsize == 0 || fsize > MAX_FILE_SIZE )
+
+		if ( fsize == 0 || fsize > MAX_FILE_SIZE )
 		{
 			std::cerr << "File " << *it << " has zero size or is "
-				"too large to be processed with bin2res." <<
-								std::endl;
+				  "too large to be processed with bin2res." <<
+				  std::endl;
 		}
+
 		char * data = new char[fsize];
 		f.read( data, fsize );
 		embed * e = new embed;
 		e->size = fsize;
-		if( it->rfind( '/' ) != std::string::npos )
+
+		if ( it->rfind( '/' ) != std::string::npos )
 		{
 			e->name = std::string( it->c_str() +
-							it->rfind( '/' ) + 1 );
+					       it->rfind( '/' ) + 1 );
 		}
 		else
 		{
 			e->name = *it;
 		}
+
 		e->cname = convertFileNameToCIdentifier( e->name );
 		embedded_data.push_back( e );
 		std::string s;
 		std::cout << "static const unsigned char " << e->cname <<
-								"_data[] = {";
+			  "_data[] = {";
 		embedData( data, fsize, std::cout );
 		std::cout << std::endl << "};" << std::endl << std::endl;
 		delete[] data;
 	}
 
-	if( embedded_data.size() > 0 )
+	if ( embedded_data.size() > 0 )
 	{
 		std::cout << "static const unsigned char dummy_data[] ="
-					"{ 0x00 };" << std::endl << std::endl;
+			  "{ 0x00 };" << std::endl << std::endl;
```

Comment:
I tried a few things, seems like astyle always try to do the best visual alignement and that it's not possible to get completely rid of spaces. I managed to get it doing it a little better, but there's still some spaces, see following commented code.